source/_integrations/lgthinq.markdown

```diff
@@ -0,0 +1,188 @@
+---
+title: LG ThinQ
+description: Setup for LG ThinQ Integration.
+featured: true
+ha_category:
+  - Hub
+ha_release: 2024.7.3
+ha_iot_class: Cloud Push
+ha_code_owners:
+  - '@LG-ThinQ-Integration'
+ha_config_flow: true
+ha_domain: lgthinq
+ha_platforms:
+  - binary_sensor
+  - button
+  - climate
+  - fan
+  - number
+  - select
+  - sensor
+  - switch
+  - text
+  - vacuum
+  - event
```

Comment:
It is not possible to have all platforms in the initial PR.

Please limits PRs to a single platform at the time.

Reply:
@frenck 
The initial code PR will have a single platform. Do you mean that I need to make a documentation PR for every code PR with a single platform?

Reply:
Yup. Every core PR affecting documentation needs its own documentation PR linked.

Reply:
@frenck 
I limited this PR to a single platform.